tests/unit/resources/test_fcsfile.py

```diff
@@ -159,3 +159,54 @@ def test_get_events_save_kwargs(ENDPOINT_BASE, client, fcs_files):
 
     # Then:
     assert file._events_kwargs == {"compensatedQ": False, "seed": 10}
+
+
+@responses.activate
+def test_create_from_s3(ENDPOINT_BASE, client, fcs_files):
+    s3_access_key = "AKIATAAFU4EEA2CBDGRM"
+    s3_secret_key = "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"
```

Comment:
The bucket contents aren't sensitive, but let's not open ourselves up to a billing attack (downloading bucket contents tons of times or storing tons of stuff). Can you replace these with the env vars `S3_ACCESS_KEY` and `S3_SECRET_KEY` and add this to push.yml for the steps that need it?
```
env:
  S3_ACCESS_KEY: ${{ secrets.S3_ACCESS_KEY}}
  S3_SECRET_KEY: ${{ secrets.S3_SECRET_KEY}}
```

I'll change the credentials on the bucket shortly.

Reply:
This is still pending.

Reply:
It is? It `push.yml` or in `test_fcsfile.py`?